prod-sample/sample-modules/frank-lloyd-root/0.0.0/src/config.js

```diff
@@ -18,6 +18,7 @@ import csp from './csp';
 import createFrankLikeFetch from './createFrankLikeFetch';
 
 export default {
+  eventLoopLagThreshold: Infinity,
```

Comment:
do these not work with the default ?

Reply:
This prevents the circuit from opening during the integration tests